packages/cds-annotation-parser/test/data/invalid/expression/orphaned operand/ast.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
We shouldn't have whitespace in folder names.

Reply:
renamed